libtenzir/builtins/connectors/tcp.cpp

```diff
@@ -148,6 +148,10 @@ class tcp_bridge {
       [this](atom::stop) -> caf::result<void> {
         return stop();
       },
+      [this](const caf::exit_msg& msg) -> caf::result<void> {
+        TENZIR_TRACE("[tcp_bridge] exit msg reason: {}", msg.reason);
+        return stop();
+      },
```

Comment:
We should keep this consistent. I know that there are quite a few other `[…]` trace logs elsewhere, but we should not put them into rather random points (in particular as `TRACE`, given that this is not even compiled it most of the time). Doesn't really matter in this specific instance, but saying this more generally. I would still just remove the log for now.

```suggestion
      [this](const caf::exit_msg& msg) -> caf::result<void> {
        return stop();
      },
```